app/views/assets/_form.html.slim

```diff
@@ -1,4 +1,4 @@
-= form_for @asset, :html => { :multipart => true , :class => 'form-horizontal' } do |f|
+= form_for @asset, html: { :multipart => true , :class => 'form-horizontal' } do |f|
```

Comment:
:multipart => true , :class => 'form-horizontal'

You can get rid of the => here too